lib/modulesync/repository.rb

```diff
@@ -31,6 +31,8 @@ def remote_branch_differ?(local_branch, remote_branch)
     end
 
     def default_branch
+      return Git.default_branch(@directory) if Git.respond_to? 'default_branch'
```

Comment:
I'd leave a comment so it's easier to track in the future. And I think the other changes would be needed to please RuboCop
```suggestion
      # Requires ruby-git >= 1.17.0
      return Git.default_branch(@directory) if Git.respond_to?(:default_branch)

```

Reply:
Comment added. I noticed I should have probably used a symbol even before rubocop told me off! :)